### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by KaptenJansson**
**Description**
Update eslint
Add .eslintcache to gitignore
Update webdriver and chromedriver.

**Purpose**
Fix breakages and npm warnings

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,16 +26,19 @@
     "eslint-config-webrtc": "^1.0.0",
     "express": "^4.14.1",
     "faucet": "0.0.1",
+    "geckodriver": "1.4.0",
```
Comment (fippo) [inline package.json]: pinning should not be required... also this should pull from utils?

File: package.json [hunk 2]
```diff
@@ -26,16 +26,19 @@
...
     "grunt-htmlhint": ">=0.9.12",
     "pem": "^1.9.4",
-    "webrtc-utilities": "^1.0.0",
-    "geckodriver": "1.4.0",
-    "selenium-webdriver": "3.3.0"
+    "selenium-webdriver": "^3.4.0",
+    "webrtc-utilities": "^1.0.0"
+  },
+  "dependencies": {
+    "chromedriver": "^2.33.2"
```
Comment (fippo) [inline package.json]: should be dev-dependency (and come from utils)

File: package.json (hunk 1 above)
Comment (KaptenJansson) [reply package.json]: Its strange, when pulled from utilities, it's installed it says, but does not execute all the steps (when I do an explicit install I see it's added to node modules) and thus is not in node_modules nor in .bin but it's in the tree if I do npm ls.....

File: package.json (hunk 2 above)
Comment (KaptenJansson) [reply package.json]: right my bad from testing.

File: package.json (hunk 1 above)
Comment (KaptenJansson) [reply package.json]: Aha, I know why, utilities do not have geckodriver listed as a devDeps, just a normal deps.

Comment (fippo) [reply package.json]: if its a dep it will be puled and installed. if its a dev-dep it wont. Lets get rid of the utils and pull everything directly?

**Pull request merged**